.travis-deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Originally from https://gist.github.com/domenic/ec8b0fc8ab45f39403dd, with my
+# modifications for webpack.
+
+TARGET_BRANCH="pr3__bikeshed"
```

Comment:
I assume that this is temporary.  Is the intent to push to the branch that matched the request and push to that?